src/features/bivariate_color_manager/components/SentimentsCombinationsList/SentimentsCombinationsList.tsx

```diff
@@ -1,36 +1,57 @@
-import { nanoid } from 'nanoid';
 import clsx from 'clsx';
-import { memo } from 'react';
+import React, {
+  Children,
+  cloneElement,
+  isValidElement,
+  memo,
+  useRef,
+  useState,
+} from 'react';
+import { CSSTransition } from 'react-transition-group';
 import { joinAndCapitalizeItems, sortByKey } from '~utils/common';
 import { i18n } from '~core/localization';
 import { MiniLegend } from '~features/bivariate_color_manager/components/MiniLegend/MiniLegend';
 import { invertClusters } from '~utils/bivariate';
 import s from './SentimentsCombinationsList.module.css';
-import type { BivariateColorManagerData } from '~features/bivariate_color_manager/atoms/bivariateColorManagerResource';
+import type {
+  BivariateColorManagerData,
+  TableDataValue,
+} from '~features/bivariate_color_manager/atoms/bivariateColorManagerResource';
 import type { BivariateLegend } from '~core/logical_layers/types/legends';
+import type { CSSTransitionProps } from 'react-transition-group/CSSTransition';
 
 type Row = {
   key: string;
   maps: number;
   verticalLabel: string;
   horizontalLabel: string;
   legend?: BivariateLegend;
-  id: string;
 };
 
 type SentimentsCombinationsListProps = {
   data: BivariateColorManagerData;
 };
 
 const sortDescendingByMaps = sortByKey<Row>('maps', 'desc');
+const sortDescendingByQuality = sortByKey<TableDataValue>('quality', 'desc');
 
 const convertDirectionsArrayToLabel = (directions: string[][]) => {
   const [from = '', to = ''] = directions;
   return `${joinAndCapitalizeItems(from)} → ${joinAndCapitalizeItems(to)}`;
 };
 
+const fadeClassNames = {
+  enter: clsx(s.fadeEnter),
+  enterActive: clsx(s.fadeEnterActive),
+  exit: clsx(s.fadeExit),
+  exitActive: clsx(s.fadeExitActive),
```

Comment:
Record `enter: clsx(s.fadeEnter)` not make any sense, is the same as `enter: s.fadeEnter`
The reason why we use clsx is that you can write dynamic, conditional styles - like
```
className={clsx({ [s.fadeEnter]: isSomeBooleanProps, [s.fadeEnterActive]:  isSomeBooleanStateValue })}
```
also in easy way for join multiple classes: `clsx(s.classA, s.classB)` instead of `[s.classA, s.classB].join(' ')`

Reply:
oh, sure
thanks for this catch!